src/types/wallet.ts

```diff
@@ -1,11 +1,15 @@
 import { WalletState } from '@web3-onboard/core';
 import { Account } from '@web3-onboard/core/dist/types';
-import { providers } from 'ethers';
+
+import RetryRpcProvider from '~redux/sagas/wallet/RetryProvider';
+import { CustomEIP1193Provider } from '~redux/sagas/wallet/ganacheModule';
 
 export type ColonyWallet = BasicWallet | FullWallet;
 
 export interface FullWallet extends WalletState, Account {
-  ethersProvider: providers.Provider | null;
+  // @ts-ignore
```

Comment:
Why do we need that? It's not showing any errors when I remove it

Reply:
Can't remember why this is here. I'll investigate.

Reply:
Oh, I do remember now. This was added before I added the types to the `RetryProvider` class and it kept complaining, and you are correct that it's pointless now.

Have removed it.